### DISCUSSION TITLE
Sync Schema [Jun 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5955] [Discord Threads]
> 🙋‍♂️ **thomastraum**

**Comment by o1lab** [comment]
> 🙋‍♂️ **thomastraum**  
#### here is a screenshot  
![image0](https://cdn.discordapp.com/attachments/1108671779980984320/1108672179316469800/image.png)  
#  
> 🙋‍♂️ **thomastraum**  
#### ```python   
####    
#### (0, 'id', 'INTEGER', 1, None, 1)   
#### (1, 'created_at', 'datetime', 0, 'CURRENT_TIMESTAMP', 0)   
#### (2, 'updated_at', 'datetime', 0, 'CURRENT_TIMESTAMP', 0)   
#### (3, 'symbol', 'varchar', 0, None, 0)   
#### (4, 'name', 'varchar', 0, None, 0)   
####    
#### ```  
#  
> 🙋‍♂️ **wkw**  
#### formula is auto generated - it won't store in db  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1108671779980984320)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
